backend/config/config.go

```diff
@@ -127,8 +128,13 @@ func (backend Backend) CoinActive(code coin.Code) bool {
 
 // AppConfig holds the whole app configuration.
 type AppConfig struct {
-	Backend  Backend     `json:"backend"`
-	Frontend interface{} `json:"frontend"`
+	Backend Backend `json:"backend"`
+	// Frontend structure is owned by frontend/web/src/{i18n,config},
+	// hence empty interface values. However, one key is shared with
+	// the backend: userLanguage. When a stored app config is missing,
+	// the backend attempts to parse user preferred UI language and
+	// sends it to the frontend in a default config.
```

Comment:
What about graduating the user language to a backend config field?

The Backend configs are meant for fields that the backend will access. Frontend is for frontend-only config and opaque to the backend.

Reply:
I thought about it The backend is always stored on disk though, and I wanted something runtime-only. So, added a new struct field called "volatile". Let's see how it goes.

Reply:
If the field is a pointer, it can be nil, which could mean "optional / not set / not defined", like is usual in Go<->Json, and still be in the Backend struct.

Adding a whole struct full of fields that are "optional" seems a bit off when you can have `null` for the optional fields instead. WDYT?

Reply:
Sure, a nil field in the backend struct would also work but it's really a hack at that point. :)
I just don't like mixing things. To me, the backend config is for all background things, nothing to do with the native language. So, something in between clearly separated it.

Reply:
Take a look at the new version. Tell me what you think.

Reply:
All trouble is because AppConfig is used for both saving to disk and communicating with the frontend.

Reply:
After talking elsewhere, settled on a separate `GET /native-locale` endpoint instead of the hacky config fields stuff. Will change it.

Reply:
Done.